chapter-11/end/Engine/Objects/BaseGameObject.cs

```diff
@@ -12,6 +12,8 @@ public class BaseGameObject
         protected Texture2D _boundingBoxTexture;
 
         protected Vector2 _position = Vector2.One;
+        protected float _angle;
```

Comment:
Nice refactoring moving this to the base class

Reply:
I wish I could take credit. The tech reviewer caught this.